common/src/main/java/org/jboss/hal/testsuite/page/BasePage.java

```diff
@@ -224,6 +225,17 @@ public NotificationCenterFragment getNotificationArea() {
         return notification;
     }
 
+    /**
+     * @return returns area display message as popup element in center of page, Area is closed after few seconds automatically usually.
+     */
+    public AlertFragment getAlertArea() {
+        WebElement element = browser.findElement(By.cssSelector(AlertFragment.ROOT_SELECTOR));
```

Comment:
I meant something like `Graphene.waitGui().until().element(By.cssSelector(AlertFragment.ROOT_SELECTOR)).is().visible();`